Antidote/PauseCallTableViewCell.m

```diff
@@ -82,8 +83,9 @@ - (void)installConstraints
 {
     [self.endCallButton makeConstraints:^(MASConstraintMaker *make) {
         make.right.equalTo(self.contentView.rightMargin);
-        make.height.equalTo(self.contentView);
+        make.height.equalTo(kEndCallButtonWidth);
         make.width.equalTo(kEndCallButtonWidth);
```

Comment:
This is not `Width` anymore, but `Size`. Also you can replace these two with `make.size.equalTo(kEndCallButtonSize);`